internal/weldr/api.go

```diff
@@ -1757,7 +1757,14 @@ func (api *API) fetchPackageList() (rpmmd.PackageList, error) {
 	return packages, err
 }
 
-func (api *API) depsolveBlueprint(bp *blueprint.Blueprint, clean bool) ([]rpmmd.PackageSpec, map[string]string, error) {
+func (api *API) depsolveBlueprint(bp *blueprint.Blueprint, outputType, arch string, clean bool) ([]rpmmd.PackageSpec, []rpmmd.PackageSpec, map[string]string, error) {
```

Comment:
I'd prefer if new code used the types from `internal/common` and variable names reflected what is written in the README file. So I'd suggest to replace `outputType` with `imageType`.